src/BookYourShow.Api.Test/ReviewsUnitTestController.cs

```diff
@@ -0,0 +1,239 @@
+            Assert.IsType<NotFoundResult>(notFoundResult.Result);
+        }
+
+
+        [Fact]
+
+        public void AddingReviews()
+        {
+            //ok response
+            //arrange
+            var reviews = new List<Reviews>();
+            var MockRepo = new Mock<IReviewsrepo>();
+
+            MockRepo.Setup(r => r.AddReviews(It.IsAny<Reviews>())).ReturnsAsync(4);
+
+
+            var controller = new ReviewsController(MockRepo.Object);
+            var newValidItem = new Reviews()
+            {
+                MovieId = 2,
+                Comments = "Amazing Movie"
+            };
+
+            //act
+            var result = controller.AddReviews(newValidItem);
+
+            //assert
+
+            Assert.IsType<OkObjectResult>(result.Result);
+
+
+            //BadRequest
+
+            //arrange
+
+
+            var newInValidItem = new Reviews()
+            {
+                Comments = "Amazing Movie"
+            };
+
+            controller.ModelState.AddModelError("MovieId", "MovieId is required");
+
+
+            //act 
+            var badrequestResult = controller.AddReviews(newInValidItem);
+
+            //assert
+            Assert.IsType<BadRequestResult>(badrequestResult.Result);
+
+        }
+
+        [Fact]
+        public void UpdatingReviews()
+        {
+            // OK response
+            //arrange
+            var reviews = new List<Reviews>();
+            var MockRepo = new Mock<IReviewsrepo>();
+
+            var expectedResponse = new Reviews()
+            {
+                ReviewId = 2,
+                MovieId = 1,
+                Comments = "Good"
+            };
+            MockRepo.Setup(r => r.UpdateReviews(It.IsAny<Reviews>())).ReturnsAsync(expectedResponse);
+
+
+            var controller = new ReviewsController(MockRepo.Object);
+            var newValidItem = new Reviews()
+            {
+                ReviewId = 2,
+                MovieId = 1,
+                Comments = "Good"
+            };
+
+            //act
+            var result = controller.UpdateReviews(newValidItem);
+
+            //assert
+
+            var actualResponse = Assert.IsType<OkObjectResult>(result.Result);
+            var actualResult = Assert.IsType<Reviews>(actualResponse.Value);
+            Assert.Equal(expectedResponse.Comments, actualResult.Comments);
+
+            //NotFound
+
+            //arrange
+            Reviews notfoundItem = null;
+
+            MockRepo.Setup(r => r.UpdateReviews(It.IsAny<Reviews>())).ReturnsAsync(notfoundItem);
+
+            //act
+            var controller1 = new ReviewsController(MockRepo.Object);
+
+            var newValidItem2 = new Reviews()
+            {
+                ReviewId = 100,
+                MovieId = 1,
+                Comments = "Good"
+            };
+
+            var notFoundResponse = controller1.UpdateReviews(newValidItem2);
+
+            //assert
+            Assert.IsType<NotFoundResult>(notFoundResponse.Result);
+
+
```

Comment:
remove unwanted empty lines

Reply:
I had removed unwanted empty lines and comments